contrib/babelfishpg_common/sql/upgrades/babelfish_common_helper--5.1.0--5.2.0.sql

```diff
@@ -44,5 +44,37 @@ BEGIN
 END
 $body$;
 
+CREATE OR REPLACE FUNCTION sys.int4varbinary(INT4, integer, boolean)
+RETURNS sys.BBF_VARBINARY
+AS 'babelfishpg_common', 'int4varbinary'
+LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION sys.float4varbinary(REAL, integer, boolean)
+RETURNS sys.BBF_VARBINARY
+AS 'babelfishpg_common', 'float4varbinary'
+LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+-- (INT4 AS sys.BBF_VARBINARY)
+DROP CAST (INT4 AS sys.BBF_VARBINARY);
+
+-- (REAL AS sys.BBF_VARBINARY)
+DROP CAST (REAL AS sys.BBF_VARBINARY);
+
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    CREATE CAST (INT4 AS sys.BBF_VARBINARY)
+    WITH FUNCTION sys.int4varbinary (INT4, integer, boolean) AS IMPLICIT;
+
+    CREATE CAST (REAL AS sys.BBF_VARBINARY)
+    WITH FUNCTION sys.float4varbinary (REAL, integer, boolean) AS IMPLICIT;
+EXCEPTION WHEN duplicate_object THEN
+    GET STACKED DIAGNOSTICS
+    exception_message = MESSAGE_TEXT;
+    RAISE WARNING '%', exception_message;
+END;
+$$;
```

Comment:
since we are dropping these cast just above the DO block, we do not need this exception handling